src/store/list-type-store.ts

```diff
@@ -0,0 +1,29 @@
+import { ListTypes } from 'lib/enums';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { AppThunk } from './app-thunk';
+
+export interface ListTypeStore {
+  listType: ListTypes;
+}
+
+export const initialState: ListTypeStore = {
+  listType: ListTypes.Products
+};
+
+const slice = createSlice({
+  name: 'listType',
+  initialState,
+  reducers: {
+    setListType: (state: ListTypeStore, action: PayloadAction<ListTypes>) => {
+      state.listType = action.payload;
+    }
+  }
+});
+
+export const { setListType } = slice.actions;
+
+export const reducer = slice.reducer;
+
+export const changeListType = (listType: ListTypes): AppThunk => async (dispatch, store) => {
```

Comment:
Where do you use this? As I have seen from the code, you only use the local state for the list type

Reply:
You are right, I don't, forgot about this, will take care of it